src/components/shared/navbar/Navbar.tsx

```diff
@@ -1,43 +1,34 @@
-import autobind from "autobind-decorator";
 import classNames from "classnames";
 import * as React from "react";
-
 import IconButton from "../icon-button/IconButton";
-
 import "./Navbar.scss";
 
-interface INavbarProps {
+interface NavbarProps {
   backButton?: boolean;
   title?: string;
   onBackButtonClick?: () => void;
```

Comment:
Is there ever a time that `backButton = true` and `onBackButtonClick = undefined`?  can the existence of `onBackButtonClick` be used as the trigger to include the back button?

Reply:
Nope! Solved :tada: